replication-api-impl/src/test/java/org/codice/ditto/replication/api/impl/persistence/SiteManagerImplTest.java

```diff
@@ -48,6 +48,7 @@
   @Before
   public void setUp() {
     System.setProperty("org.codice.ddf.system.siteName", "testSite");
+    System.setProperty("org.codice.ddf.system.rootContext", "services");
```

Comment:
❗️ System properties should be cleaned up after the test has run. Use the RestoreSystemProperties JUnit rule from codice-test

Reply:
added restore annotation